src/ddprof_input.cc

```diff
@@ -56,11 +56,27 @@
 #define X_HLPK(a, b, c, d, e, f, g, h, i)                                      \
   "  -" #c ", --" #b ", (envvar: " #a ")",
 
-// Helpers for expanding the OPT_TABLE here
+namespace {
+std::string mask_value(const std::string_view value) {
+  if (value.size() != k_size_api_key) {
```

Comment:
Should we do stricter validation?  https://github.com/DataDog/dd-trace-go/blob/main/profiler/options.go#L188

I'm divided because this isn't really a feature we want to lean on, but there are so many places you can go wrong with copy+pasting things from docs/setup.  :)

Reply:
Yes! This does not make the top of the list, but it is high up there ^^

Reply:
I think changing CLI tool would help here. A lot of parameters would have stronger types and out of the box validation.
I'll check on that later.